tsdef/RemoteParticipant.d.ts

```diff
@@ -0,0 +1,39 @@
+import { NetworkQualityLevel, NetworkQualityStats } from './NetworkQuality';
+import { Participant } from './Participant';
+import { RemoteAudioTrackPublication } from './RemoteAudioTrackPublication';
+import { RemoteDataTrack } from './RemoteDataTrack';
+import { RemoteDataTrackPublication } from './RemoteDataTrackPublication';
+import { RemoteTrack } from './types';
+import { RemoteTrackPublication } from './RemoteTrackPublication';
+import { RemoteVideoTrack } from './RemoteVideoTrack';
+import { RemoteVideoTrackPublication } from './RemoteVideoTrackPublication';
+import { Track } from './Track';
+
+export namespace RemoteParticipantDataTrack {
+  type dataTracks = Map<Track.SID, RemoteDataTrackPublication>;
+}
+
+export class RemoteParticipant extends Participant {
+  audioTracks: Map<Track.SID, RemoteAudioTrackPublication>;
```

Comment:
I've moved the `dataTracks` property into a `namespace` because I'm for some reason getting this error when I have the property on the class itself. Although the issue isn't happening in `LocalParticipant`
![image](https://user-images.githubusercontent.com/43423318/97099613-39e99d80-1648-11eb-9747-4c35a1797b50.png)

Reply:
Please make this change.